projects/gui/src/newtournamentdialog.cpp

```diff
@@ -240,6 +254,39 @@ Tournament* NewTournamentDialog::createTournament(GameManager* gameManager) cons
 	t->setGamesPerEncounter(ui->m_gamesPerEncounterSpin->value());
 	t->setRoundMultiplier(ui->m_roundsSpin->value());
 
+	QString bookFileName = ui->m_bookFileEdit->text();
+	if (not bookFileName.isEmpty())
+	{
+		OpeningSuite::Format format = OpeningSuite::PgnFormat;
+		if (bookFileName.endsWith(".pgn"))
+			format = OpeningSuite::PgnFormat;
+		else if (bookFileName.endsWith(".epd"))
+			format = OpeningSuite::EpdFormat;
+		else
+			{} // assume the user knows what he is doing
+
+		OpeningSuite::Order order = OpeningSuite::SequentialOrder;
+		if (ui->m_seqOrderRadio->isChecked())
+			order = OpeningSuite::SequentialOrder;
+		else if (ui->m_randomOrderRadio->isChecked())
+			order = OpeningSuite::RandomOrder;
+		else
+			return 0;
```

Comment:
Can the code even reach this line? If it can, the UI logic should probably be reworked so that one of the radio buttons must always be checked.